index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+    
+
+    -->
+    <div class="container">
+      <div>
+        <h1 id="title" class="center">freeCodeCamp Survey Form</h1>
+        <p id = "description" class="center">Thank you for taking the time to help us improve the platform</p>
+      </div>
```

Comment:
I think it would be better if you use a semantic HTML tag for this div.
This is the header of the page so header tag would be more appropriate.